docs/leaf/key-concepts/leaf-mtg.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Pando Leaf MTG
+sidebar_position: 4
+date: 2021-08-15 22:33:07
+---
+
+Pando Leaf is built with [Mixin Trusted Group](https://developers.mixin.one/document/mainnet/mtg). Right now, there are 6 genesis nodes who serve the trades, they are:
```

Comment:
-->" ... there are 6 managerial nodes that performs administrative actions. They are: "